test/network_analysis/network_properties.jl

```diff
@@ -326,3 +326,81 @@ let
     @test Catalyst.iscomplexbalanced(rn, rates) == true 
 end
 
+### STRONG LINKAGE CLASS TESTS
+
+let
```

Comment:
Generally I try to, before each `let ... end` block in the tests, have some kind of comment what is being tested. That can be useful if someone tries to understand the tests. Especially long-term if they need to be modified for some reason.

Here I presume you just test the `stronglinkageclasses` and `terminallinkageclasses` functions? However, it might be good to state that, and also if there are some important differences between the networks. If each test just tests these functions and that is it, it does not hurt to also mention that.